lib/modules/platform/bitbucket/index.ts

```diff
@@ -952,11 +956,59 @@ export async function createPr({
         renovateUserUuid,
         pr,
       );
+      if (platformPrOptions?.bbAutoResolvePrTasks) {
+        await autoResolvePrTasks(pr);
+      }
       return pr;
     }
   }
 }
 
+async function autoResolvePrTasks(pr: Pr): Promise<void> {
+  logger.debug(`Auto resolve PR tasks in #${pr.number}`);
+  try {
+    const response = (
+      await bitbucketHttp.getJson(
+        `/2.0/repositories/${config.repository}/pullrequests/${pr.number}/tasks`,
+        { paginate: true, pagelen: 100 },
+      )
+    ).body;
+
+    logger.trace(
+      {
+        prId: pr.number,
+        listTaskRes: response,
+      },
+      'List PR tasks',
+    );
+
+    const unResolvedTasks: Array<PrTask> = UnresolvedPrTasks.parse(response);
```

Comment:
```suggestion
    const unResolvedTasks = UnresolvedPrTasks.parse(response);
```
pass this type to http request as schema, so validation is done in http layer